AlgorithmFactory/Loader.cs

```diff
@@ -289,7 +289,7 @@ private bool TryCreateILAlgorithm(string assemblyPath, out IAlgorithm algorithmI
             {
                 errorMessage = "Algorithm type name not found, or unable to resolve multiple algorithm types to a single type. Please verify algorithm type name matches the algorithm name in the configuration file and that there is one and only one class derived from QCAlgorithm.";
                 errorMessage += err.InnerException == null ? err.Message : err.InnerException.Message;
-                Log.Error($"Loader.TryCreateILAlgorithm(): {errorMessage}");
+                Log.Error($"Loader.TryCreateILAlgorithm(): {errorMessage}\n{err}");
```

Comment:
In the line above we are already adding the exception message associated, it seems to me the stacktrace doesn't really add much more information? For example:
```   at QuantConnect.Algorithm.CSharp.BasicTemplateAlgorithm..ctor()
   --- End of inner exception stack trace ---
   at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean noCheck, Boolean& canBeCached, RuntimeMethodHandleInternal& ctor, Boolean& bNeedSecurityCheck)
   at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic)
   at System.RuntimeType.CreateInstanceImpl(BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes, StackCrawlMark& stackMark)
   at System.Activator.CreateInstance(Type type, BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes)
   at System.Reflection.Assembly.CreateInstance(String typeName, Boolean ignoreCase, BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes)
   at System.Reflection.Assembly.CreateInstance(String typeName, Boolean ignoreCase)
   at QuantConnect.AlgorithmFactory.Loader.TryCreateILAlgorithm(String assemblyPath, IAlgorithm& algorithmInstance, String& errorMessage) in D:\QuantConnect\MyLean\Lean\AlgorithmFactory\Loader.cs:line 274
```

Reply:
It depends how many layers there are from the inner exception.

If there's one child that will include it, but it doesn't include every nested exception.